pycbc/conversions.py

```diff
@@ -395,6 +425,107 @@ def secondary_spin(mass1, mass2, spin1, spin2):
     ss[mask] = spin1[mask]
     return _formatreturn(ss)
 
+def primary_xi(spinx, spiny):
+    """Returns the effective precession spin argument for the larger mass.
+    """
+    return chi_perp_from_spin1x_spin1y(spinx, spiny)
```

Comment:
Make this take mass1, mass2, spin1x, spin1y, spin2x, and spin2y. I know you only need the spins for xi1, but the `primary` label indicates that the returned values are guaranteed to be for the larger object. If you make it take mass1, mass2, you can guarantee that that is true for an array of values in which mass1 is not always greater than mass2 (as might be the case in a PE run).

That said, it is nice having a simple function that return xi1 given two spins. So, rename this function `xi1_from_spinx_spiny`, and make it explicit in the doc string that it is assumed the spins are from the larger body. Then add a function `primary_xi` that takes in the masses and spins of both objects, uses the `primay_spin` function to pull out the spinx and y of the larger body, then passes it to `xi1_from_spinx_spiny`.